kubert/src/lease.rs

```diff
@@ -125,6 +126,8 @@ impl Claim {
 
 impl LeaseManager {
     const DEFAULT_FIELD_MANAGER: &'static str = "kubert";
+    const DEFAULT_MIN_BACKOFF: Duration = Duration::from_secs(2);
```

Comment:
should we consider a shorted min backoff? I think we'd want the first retry to be almost immediate so that we recover from transient errors as quickly as possible.